Hypeclass/API/DanceClassManager.swift

```diff
@@ -18,4 +18,32 @@ class DanceClassManager {
             print("Error writing dance class to Firestore: \(error)")
         }
     }
+    
+    func requestDanceClassByDancerID(id: String) async throws -> [DanceClass]? {
```

Comment:
호출할때 id 값을 명시하는게 좋을까요? 
_ id: String 이 나을지 id: String  이 나을지 고민입니다! 

저는 함수 사용할때 항상 편하게 사용하기 위해 _ id: String 으로 사용하곤 하는데 팀 컨벤션을 한번 이야기 해보면 좋을것 같아요!

Reply:
넵 내일 한 번 얘기해 보아요 😊

Reply:
파라미터로 DancerID 를 빼면 함수 이름이 더 직관적일 것 같아요!
```suggestion
    func requestDanceClassBy(dancerID id: String) async throws -> [DanceClass]? {
```

Reply:
그리고 해당 메서드가 어레이를 반환하는데 requestDanceClass 네이밍으로 둬도 괜찮을지 이야기해보면 좋을 것 같습니다 ㅎㅎ

Reply:
> 파라미터로 DancerID 를 빼면 함수 이름이 더 직관적일 것 같아요!

External parameter name을 쓰니까 간단해지네요! 반영했습니다 :)